lib/rubygems/commands/update_command.rb

```diff
@@ -328,12 +328,8 @@ def oldest_supported_version
     @oldest_supported_version ||=
       if Gem.ruby_version > Gem::Version.new("3.1.a")
         Gem::Version.new("3.3.3")
-      elsif Gem.ruby_version > Gem::Version.new("3.0.a")
-        Gem::Version.new("3.2.3")
-      elsif Gem.ruby_version > Gem::Version.new("2.7.a")
-        Gem::Version.new("3.1.2")
       else
-        Gem::Version.new("3.0.1")
+        Gem::Version.new("3.2.3")
```

Comment:
Above you said 3.1.0 shipped with ruby 3.0, but was it actually 3.2.3?

Reply:
Yes, my bad, this is correct, the gemspec info is wrong. Will correct it.